ui/layouts/OwnerApp.tsx

```diff
@@ -24,6 +26,14 @@ export const OwnerApp: React.FC<Props> = ({ children, title }) => {
   const { data: companies } = useCompanies()
   const { data: owner } = useOwner()
 
+  const [hint, setHint] = React.useState(() => {
+    return localStorage.getItem('hintclosed') !== '1')
```

Comment:
Did an oopsie in my code review, sorry!

```suggestion
    return localStorage.getItem('hintclosed') !== '1'
```